code/modules/mob/living/living_defense.dm

```diff
@@ -261,21 +261,21 @@
 				else
 					visible_message("<span class='danger'>[user] has grabbed [src] aggressively!</span>", \
 									"<span class='userdanger'>[user] has grabbed you aggressively!</span>")
-					drop_all_held_items()
+					update_mobility()
```

Comment:
you sure drop_all_held_items() should be replaced by update_mobility()?

Reply:
yeah that drops items for you 

	//Handle update-effects.
	if(!(mobility_flags & MOBILITY_HOLD))
		drop_all_held_items()